DependencyInjection/Configuration.php

```diff
@@ -85,6 +85,9 @@ public function getConfigTreeBuilder()
                         ->scalarNode('enable_sibling_provider')->defaultValue(false)->end()
                     ->end()
                 ->end()
+                ->arrayNode('content_listener')
+                    ->canBeDisabled()
```

Comment:
This is actually how I originally wrote it (booleanNode) however my thought was we may want to allow specifying a service_id to use instead of the out of the box content listener so you could provide your own content listener and not have to rewrite the alternate locale provider portion. I'm perfectly happy to switch it to a booleanNode, just figured I'd provide my rational first